sync/gh.py

```diff
@@ -126,12 +126,11 @@ def approve_pull(self, pr_id):
         )
 
     def status_checks_pass(self, pr_id, exclude=None):
+        c = self.get_combined_status(pr_id)
         if exclude is None:
-            exclude = set()
-        statuses = {item.context: item for item in self.get_combined_statuses(pr_id)
-                    if item.context not in exclude}
-        non_success = {context: item for context, item in statuses.iteritems()
-                       if item.state != "success"}
+            return c.state == "success", [item for item in c.statuses if item.state != "success"]
+        statuses = [item for item in c.statuses if item.context not in exclude]
+        non_success = [item for item in statuses if item.state != "success"]
```

Comment:
Isn't this just `non_success = [item for item in statuses if item.context not in exclude and item.state != "success"]`